### ISSUE TITLE
zeroctl rebalance command

### ISSUE THREAD

**Issue by GlenDC** [zero-os/0-Disk#301]
Part of the self-healing as described in https://github.com/zero-os/0-orchestrator/blob/master/specs/selfhealing/storage.md.

TODO:
 
+ describe parameters and how the command would look like;
+ describe what the command would do and to which storage types it would apply (if specific at all);

**Comment by GlenDC** [comment]
This issue depends on #303 being implemented.

The `zeroctl rebalance` command and re-spreading nbdserver algorithm will have to make use of the backup storage cluster to recover the shards lost from the lost storage server, as part of the process of re-spreading that data onto available primary storage servers.

**Comment by GlenDC** [comment]
For now I don't think this command will be required so I'm closing this as wontfix.
